### PR TITLE
Freemabd1/debug create review

### PR DESCRIPTION
**Pull request by freemabd1**

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java [hunk 1]
```diff
@@ -160,11 +190,31 @@
...
+        stopWatch = new StopWatch();
+        stopWatch.start();
+        log.log(Level.INFO, "starting saveCohortReview()");
         cohortReviewService.saveCohortReview(cohortReview);
+        stopWatch.stop();
+        log.log(Level.INFO, "saveCohortReview() time:" + stopWatch.getTotalTimeSeconds());
+        stopWatch = new StopWatch();
+        stopWatch.start();
+        log.log(Level.INFO, "starting saveParticipantCohortStatuses()");
         cohortReviewService.saveParticipantCohortStatuses(participantCohortStatuses);
```
Comment (danqrodney) [inline api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java]: Just a note: doing this in a separate transaction from saveCohortReview() means we could wind up in a situation where we have a review with no participant cohort statuses in it. 

Once we figure out how to address the performance bottleneck, we may need to figure out how to either write the status rows in the same transaction or handle this case gracefully.

File: api/src/main/resources/application.properties [hunk 2]
```diff
@@ -5,6 +5,11 @@
 #spring.jpa.properties.hibernate.format_sql=true
 #spring.jpa.properties.hibernate.type=trace
 
+#Hibernate batching properties
```
Comment (danqrodney) [inline api/src/main/resources/application.properties]: Did you want to turn on SQL logging above?

File: api/src/main/java/org/pmiops/workbench/cohortreview/CohortReviewServiceImpl.java [hunk 3]
```diff
@@ -85,8 +97,18 @@ public CohortReview saveCohortReview(CohortReview cohortReview) {
...
     @Override
+    @Transactional
     public Iterable<ParticipantCohortStatus> saveParticipantCohortStatuses(List<ParticipantCohortStatus> participantCohortStatuses) {
-        return participantCohortStatusDao.save(participantCohortStatuses);
+        int i = 0;
+        for (ParticipantCohortStatus participantCohortStatus : participantCohortStatuses) {
+            entityManager.persist(participantCohortStatus);
+            i++;
+            if (i % batchSize == 0) {
+                entityManager.flush();
```
Comment (danqrodney) [inline api/src/main/java/org/pmiops/workbench/cohortreview/CohortReviewServiceImpl.java]: You might want to add a log statement here.

File: api/src/main/resources/application.properties (hunk 2 above)
Comment (freemabd1) [reply api/src/main/resources/application.properties]: We can but it slows things down big time. I want to see if batching gets anything first. Then I can try something else.

File: api/src/main/java/org/pmiops/workbench/cohortreview/CohortReviewServiceImpl.java (hunk 3 above)
Comment (freemabd1) [reply api/src/main/java/org/pmiops/workbench/cohortreview/CohortReviewServiceImpl.java]: Done.

File: api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java (hunk 1 above)
Comment (freemabd1) [reply api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java]: Agreed.

**Pull request merged**